api/envoy/extensions/filters/udp/udp_proxy/v3/udp_proxy.proto

```diff
@@ -33,6 +33,15 @@ message UdpProxyConfig {
 
       // The source IP will be used to compute the hash used by hash-based load balancing algorithms.
       bool source_ip = 1 [(validate.rules).bool = {const: true}];
+
+      // A given key will be used to compute the hash used by hash-based load balancing algorithms.
+      // In certain cases there is a need to direct different UDP streams jointly towards the selected set of endpoints.
+      // A possible use-case is VoIP telephony,
```

Comment:
nit: The formatting is funny though. I guess no need to put a new line after every punctuation mark.

Reply:
@rojkov BTW I guess you must approve one more time due to my latest changes. After your approval, Matt has to approve as well, right?